fuseops/ops.go

```diff
@@ -646,6 +648,8 @@ type OpenFileOp struct {
 	// advance, for example, because contents are generated on the fly.
 	UseDirectIO bool
 
+	OpenFlags *fusekernel.OpenFlags
```

Comment:
I made this a pointer, so we don't default to `OpenReadOnly` (which is 0). Let me know if you have feedback.